spek-api/src/main/kotlin/org/jetbrains/spek/api/IgnoreSpek.kt

```diff
@@ -0,0 +1,5 @@
+package org.jetbrains.spek.api
+
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.CLASS)
+annotation class IgnoreSpek
```

Comment:
Java has this as well :) Not sure if someone will have problems importing multiple `@Ignore` into same class